lib/irc/ConnectionInstance.js

```diff
@@ -172,6 +172,10 @@ ConnectionInstance.prototype._listenForErrors = function() {
                 return; // don't disconnect for these error codes.
             }
         }
+        if (err && err.command && err.command === "err_yourebannedcreep") {
```

Comment:
I'm not sure a truthy check of `err.command` is necessary here given the `===` check with the string.